conn.go

```diff
@@ -130,6 +132,58 @@ func (cc *Conn) netlinkConnUnderLock() (*netlink.Conn, netlinkCloser, error) {
 	return nlconn, func() error { return nlconn.Close() }, nil
 }
 
+func (cc *Conn) receiveAckAware(nlconn *netlink.Conn, sentMsgFlags netlink.HeaderFlags) ([]netlink.Message, error) {
+	if nlconn == nil {
+		return nil, errors.New("netlink conn is not initialized")
+	}
+
+	// first receive will be the message that we expect
+	reply, err := nlconn.Receive()
+	if err != nil {
+		return nil, err
+	}
+
+	if !cc.lasting {
+		// in non lasting connection there are no acks being sent,
```

Comment:
I agree that a non-lasting connection cannot be confused by out of sync acks, but shouldn’t we still read the acks so that we do error checking (line 174 below), even if we open a new connection afterwards?

Reply:
Agreed.